tests/antithesis/Makefile

```diff
@@ -22,6 +22,7 @@ antithesis-build-etcd:
 	cd $(BUILDDIR) && make build
```

Comment:
`antithesis-build-etcd` and `antithesis-build-etcd-create-worktree` can be removed

Reply:
Removed.